frontend/src/components/upload.ts

```diff
@@ -29,7 +29,8 @@ type UploadInfo = UploadSuccess | UploadProgress | UploadFailure;
         <div class="d-flex justify-content-between">
             <div>
                 <label :for="uploadId"><div class="btn btn-primary">Upload a file</div></label>
-                <input class="invisible" :id="uploadId" type="file" @change="handleFiles($event)">
+                <input class="invisible" :id="uploadId" type="file"
+                       @change="handleFiles($event)" :accept="acceptedFileTypes">
```

Comment:
was acceptedFileTypes missing earlier?

Reply:
Yes. It doesn't properly filter on Firefox (all files included) but works on Safari and Chrome